### DISCUSSION TITLE
UI Bugs when using scroller wheel on Chrome

### DISCUSSION THREAD

**Discussion by paanjoe** [supabase/supabase#18085] [Contribute to Supabase]
# Bug report

- [Yes] I confirm this is a bug with Supabase, not with my own application. 
- [Yes] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

There is 1 UI bugs when I try to scroll down to of the page. I'm using Logitech MX Master 3 Mouse. When I use my scroller, it went down all the way until the UI is broken. This behaviour is already like a months.

I have tested on:
1. Windows Laptop
2. MacOS Laptop

Browser:
1. Only happen in Google Chrome

Please see the attached video

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Please use Google Chrome
2. Open your database
3. Select any table
4. Click on any rows
5. Scroll all the way down using 'scroller wheel' on your mouse all the way down
6. The issue will be reproduced as per video.

## Expected behavior
The scroll does not scroll offset from the table UI.

## Screenshots

https://github.com/supabase/supabase/assets/38974707/33bb77a6-a48f-49e6-b157-9e36ff6c2cd6

See video attached.

## System information

- OS: macOS & Windows 11
- Browser: Google Chrome - Version 117.0.5938.149 (Official Build) (x86_64)
- Version of supabase-js: Not Applicable
- Version of Node.js: Not Applicable

**Comment by saltcod** [comment]
Very strange. 

Anyone else able to reproduce this one? 
Does it happen when that mouse is disconnected?

**Comment by paanjoe** [comment]
> Very strange. 
> 
> 
> 
> Anyone else able to reproduce this one? 
> 
> Does it happen when that mouse is disconnected? 

Indeed. I have tested this with my Lead laptop with the same mouse. It does happen.

Nope, my mouse is not disconnected and fully functioning. It is weird and I thought that it might be some browser issue but it has been a month like this.

**Reply by silentworks** [reply]
I've tested this on Windows 11 with my MX Master 3 mouse on Chrome 118.0.5993.71 as this is the latest build and don't have the issue shown in your screen recording. Please try in incognito mode with all extensions disabled just to make sure it isn't a chrome extension causing the issue.
